docs/adr/0035-refactor-payments-frontend.md

```diff
@@ -0,0 +1,101 @@
+# Refactor Subscription Platform frontend
+
+- Status: proposed
+- Deciders: Ben Bangert, Lisa Chan, Barry Chen
+- Date: 2023-03-07
+
+## Context and Problem Statement
+
+The Subscription Platform frontend has become overly complex, making it inflexible and difficult to work with. This has resulted in longer development times on new features and in some cases left us unable to easily accommodate requests from relying parties.
+
+Additionally, as the app has grown, so have the number of network calls causing a reduction in app performance. The aim is to refactor the app to have one network call for each action.
+
+Finally, the frontend includes some business and application logic, which has made components and parts of the app unnecessarily complex. As a result the frontend is less flexible, and harder to change and maintain. To solve this, the Subscription Platform team wants to move all business and application logic to the backend.
+
+To address the problems briefly discussed above, and to meet current and future relying party and feature requests, refactoring of the existing Subscription Platform is necessary.
+
+## Decision Drivers
+
+- Faster development - This applies to both the refactoring necessary for SubPlat 3.0, as well as future relying party and feature requests
```

Comment:
Might make a note about what "SubPlat 3.0" is since reading this out of context might be confusing, even if it's just "dubbed SubPlat 3.0" at the end of your last sentence under the context & problem statement.